### ISSUE TITLE
Dev dependencies not in dev

### ISSUE THREAD

**Issue by skjnldsv** [nextcloud/notes#707]
Some deps should be dev deps, not regular ones :)
- `@nextcloud/webpack-vue-config`
- `stylelint-webpack-plugin`

```json
"dependencies": {
		"@nextcloud/axios": "^1.6.0",
		"@nextcloud/dialogs": "^3.1.2",
		"@nextcloud/event-bus": "^1.2.0",
		"@nextcloud/moment": "^1.1.1",
		"@nextcloud/router": "^1.2.0",
		"@nextcloud/vue": "^3.8.0",
		"@nextcloud/vue-dashboard": "^2.0.0",
		"@nextcloud/webpack-vue-config": "^4.0.1",
		"diff": "^5.0.0",
		"easymde": "^2.14.0",
		"markdown-it": "^12.0.4",
		"stylelint-webpack-plugin": "^2.1.1",
		"vue": "^2.6.12",
		"vue-fragment": "1.5.1",
		"vue-material-design-icons": "^4.11.0",
		"vue-observe-visibility": "^1.0.0",
		"vue-router": "^3.5.1",
		"vuex": "^3.6.2"
	},
```

cc @korelstar

**Comment by korelstar** [comment]
Yep, saw this already, too. Will fix it soon.

**Comment by korelstar** [comment]
Yep, saw this already, too. Will fix it soon and I will switch to `@nextcloud/babel-config`. Btw: will be there a non-beta release, soon?

**Comment by skjnldsv** [comment]
yes, but it will most likely be the same content

**Comment by korelstar** [comment]
Fixed by #709
